swal-forms.css

```diff
@@ -1,7 +1,7 @@
 .swal-form {
   padding-top: 3%;
-  overflow: scroll;
-  height: 300px;
+  overflow: auto;
```

Comment:
this change didn't interfere in swal-forms mechanics, it just made the alert windows look nicier(in my opinion). So, you can ignore this.

Here is the screenshot:
![image](https://cloud.githubusercontent.com/assets/8538337/12534253/74049b0a-c22e-11e5-919d-73a3280fd1d2.png)